Makefile

```diff
@@ -6,8 +6,7 @@ clean::  ## Remove generated files
 
 .PHONY: all clean
 
-
-# -- Test ---------------------------------------------------------------------
+# -- Test ----
```

Comment:
The commit message says "we" are converging to a common way...". Are we? when did that happen?

FTR, it looks like every repo now has a different form of these separators. One has a triple-dash at the start and four after. This is a double-dash and four after. Some others have the full line like this one.

Happy to LGTM this if it is what you want, but I just wanted to say I did not see this exact style in any other foxygoat repo.

Reply:
ok, so let's converge - what should it be?

Reply:
(I actually wanted this to be triple before and after)

Reply:
looks like you missed. ;-) 3 + 3 is fine (i think - but i reserve the right to change my mind)

Reply:
I've tried again.